scalameter-core/src/main/scala/org/scalameter/PicklerBasedKey.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter
+
+import org.scalameter.picklers.Pickler
+
+
+/* A mixin for keys whose values require pickler based serialization. */
+trait PicklerBasedKey[T] {
+  /* Name of key which will be serialized. */
```

Comment:
Nit: use `that` without a comma when the explanation is a part of the sentence that cannot be left out without changing the meaning. Use `which` with a comma to add a non-crucial extension, which can be left out.

Not very important, I'm just being nitpicky :)